client/webserver/site/src/js/markets.ts

```diff
@@ -243,10 +243,15 @@ export default class MarketsPage extends BasePage {
       bind(quoteIcons.sleeping, 'click', () => { this.showOpen(this.market.quote, this.walletUnlocked) })
       bind(baseIcons.locked, 'click', () => { this.showOpen(this.market.base, this.walletUnlocked) })
       bind(quoteIcons.locked, 'click', () => { this.showOpen(this.market.quote, this.walletUnlocked) })
+      bind(baseIcons.disabled, 'click', () => { this.showToggleWalletStatus(this.market.base) })
+      bind(quoteIcons.disabled, 'click', () => { this.showToggleWalletStatus(this.market.quote) })
```

Comment:
Do we need this functionality on the markets view?

Reply:
I think so. Just like every other wallet action like unlocking etc... Plus, It is convenient to enable wallets in one click without having to go to walletsPage -> Setting.

Reply:
I figure the "X" has to be shown, so might as well make it clickable.